cmd/gardenlet/app/gardenlet.go

```diff
@@ -353,6 +357,12 @@ func NewGardenlet(ctx context.Context, cfg *config.GardenletConfiguration) (*Gar
 		return nil, errors.New("unable to extract Gardener`s cluster identity from cluster-identity ConfigMap")
 	}
 
+	// create the certificate manager to schedule certificate rotations
+	var certificateManager *certificate.Manager
+	if cfg.GardenClientConnection.KubeconfigSecret != nil {
+		certificateManager = certificate.NewCertificateManager(cfg.GardenClientConnection, util.GetSeedName(cfg.SeedConfig), clientMap)
```

Comment:
You can just use `seedName`, right?
```suggestion
		certificateManager = certificate.NewCertificateManager(cfg.GardenClientConnection, seedName, clientMap)
```

Reply:
This is defaulting the Seed name if it is nil. See [here](https://github.com/gardener/gardener/blob/67833d9f9bbfd0b9a8ddd0cfffec6a455ead2586/pkg/gardenlet/bootstrap/util/util.go#L161-L160)

Reply:
And `seedName` is already calculated using `util.GetSeedName()` here:
https://github.com/gardener/gardener/blob/67833d9f9bbfd0b9a8ddd0cfffec6a455ead2586/pkg/gardenlet/bootstrap/bootstrap.go#L62

Reply:
thanks, makes sense